cita-chain/types/src/receipt.rs

```diff
@@ -1,18 +1,16 @@
-// Copyright 2015-2017 Parity Technologies (UK) Ltd.
```

Comment:
Merge this PR after this file is replaced?

Reply:
You mean we can not change this file to apache license ?

Reply:
You mean we can not change this file to apache license for now ?

Reply:
Not Professional.
The project muse be GPL even there is only one GPL file accoring the GPL license.
Just an advice, yes, we can. but i can not apporve.

Reply:
> The project muse be GPL even there is only one GPL file accoring to the GPL license.

Yes!